engine/execution/computation/computer/computer_test.go

```diff
@@ -303,15 +303,15 @@ func TestBlockExecutor_ExecuteBlock(t *testing.T) {
 		serviceEventA := cadence.Event{
 			EventType: &cadence.EventType{
 				Location: common.AddressLocation{
-					Address: common.Address(serviceEvents.EpochSetup.Address),
```

Comment:
I think we want to keep these `common.Address`

Reply:
I think there were some commits on reverting simple address conversion that was problematic, @turbolent do you have the context ?

Reply:
fixed them all back to use common.Address

Reply:
To add some more context: As I remember, `address.Bytes()` was removed to avoid the misuses as it truncates the leading zeros. But that API change breaks the migration tool, so temporarily reverted it in the testnet branch.

But for the master, and going forward, we can use the simple address conversion `common.Address(...)`